lib/beetle/configuration.rb

```diff
@@ -118,9 +123,6 @@ class Configuration
     # Write timeout for http requests to RabbitMQ HTTP API
     attr_accessor :rabbitmq_api_write_timeout
 
-    # Returns the port on which the Rabbit API is hosted
-    attr_accessor :api_port
```

Comment:
We removed this without replacement to simplify our change. In our codebase that's not used, and pretty many installations of rabbitmq just use the default convention of prepending a 1 to the amqp(s) port.